refaster-runner/src/test/java/tech/picnic/errorprone/refaster/runner/RefasterTest.java

```diff
@@ -167,13 +164,18 @@ void severityAssignment(
                 "    };",
                 "  }",
                 "}");
-    assertThatThrownBy(compilationTestHelper::doTest)
-        .isInstanceOf(AssertionError.class)
-        .message()
-        .satisfies(
-            message ->
-                assertThat(extractRefasterSeverities("A.java", message))
-                    .containsExactlyElementsOf(expectedSeverities));
+
+    if (expectedSeverities.isEmpty()) {
+      compilationTestHelper.doTest();
+    } else {
```

Comment:
Previously this wasn't necessary, because the `expectErrorMessage` declarations would trigger a "I didn't see anything" `AssertionError`, which would then cause the `satisfies` clause to succeed. But that's obscure.